src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.CountryDao;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
+import org.hibernate.SessionFactory;
 
 public class CountryServiceImpl implements CountryService {
+    private CountryDao countryDao;
+
+    public CountryServiceImpl(SessionFactory sessionFactory) {
+        this.countryDao = new CountryDaoImpl(sessionFactory);
+    }
+
     @Override
     public Country add(Country country) {
-        return null;
+        return countryDao.add(country);
     }
 
     @Override
     public Country get(Long id) {
-        return null;
+        return countryDao.get(id).orElse(null);
```

Comment:
not critical, but it's better newer to return null from methods
```suggestion
        return countryDao.get(id).orElseThrow(() -> new DataProcessingException("message"));
```